db/migrate/20180107082701_authlogic_to_devise.rb

```diff
@@ -0,0 +1,58 @@
+# frozen_string_literal: true
+
+class AuthlogicToDevise < ActiveRecord::Migration[5.1]
+  def self.up
+    add_column :users, :unconfirmed_email, :string, limit: 254
+    add_column :users, :reset_password_token, :string
+    add_column :users, :reset_password_sent_at, :datetime
+    add_column :users, :remember_token, :string
+    add_column :users, :remember_created_at, :datetime
+    add_column :users, :authentication_token, :string
+    add_column :users, :confirmation_token, :string, limit: 255
+    add_column :users, :confirmed_at, :timestamp
+    add_column :users, :confirmation_sent_at, :timestamp
+    execute "UPDATE users SET confirmed_at = created_at, confirmation_sent_at = created_at"
+
+    rename_column :users, :password_hash, :encrypted_password
+
+    rename_column :users, :current_login_at, :current_sign_in_at
+    rename_column :users, :last_login_at, :last_sign_in_at
+    rename_column :users, :current_login_ip, :current_sign_in_ip
+    rename_column :users, :last_login_ip, :last_sign_in_ip
+    rename_column :users, :login_count, :sign_in_count
+
+    remove_column :users, :persistence_token
```

Comment:
Will we need to encourage anyone using the atom/rss links that they need to migrate to... something?
https://github.com/plataformatec/devise/wiki/How-To:-Simple-Token-Authentication-Example suggests there not heaps there

Reply:
How is atom/rss feed access working today? Are they not covered by tests? (as all tests appear green)

Reply:
Feels a bit legacy now... time to remove them?